.github/workflows/test.yml

```diff
@@ -5,32 +5,7 @@ on:
   push:
     branches:
       - master
-  schedule:
-    # Run automatically every day
-    - cron: "0 0 * * *"
```

Comment:
I suggest creating a new workflow to run scheduled builds so that if/when it gets disabled, it doesn't stop the workflow from running on push/pull_request.

Not a great GitHub Actions behavior, IMO.